packages/cli/package.json

```diff
@@ -6,7 +6,7 @@
   "author": "thirdweb eng <eng@thirdweb.com>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=16.0.0 <19"
+    "node": ">=16.0.0"
```

Comment:
@jnsdls should this be `> 16.0.0` ? based on: https://github.com/thirdweb-dev/js/pull/1629